### PR TITLE
Custom font family

### PR DESCRIPTION
**Pull request by anhle10051996**

### CODE REVIEW

**Comment by honghoangsts** [comment]
Does all the app use the same `fontFamily`?

**Comment by anhle10051996** [comment]
> Does all the app use the same `fontFamily`?

I just add default fontFamily. If project require different fontFamily, just remove and add new fontFamily that required

**Comment by honghoangsts** [comment]
> > Does all the app use the same `fontFamily`?
> 
> I just add default fontFamily. If project require different fontFamily, just remove and add new fontFamily that required

So the effort will be double, because we need remove the old one and add another one?

File: template/android/link-assets-manifest.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    {
+      "path": "src/assets/fonts/Roboto-Thin.ttf",
+      "sha1": "711e666e7f52210ac487c7ed81a0ac68a2b52261"
+    },
+    {
+      "path": "src/assets/fonts/Roboto-ThinItalic.ttf",
+      "sha1": "75ffde032d49005faa643338eda97c311ad7d316"
+    }
+  ]
+}
```
Comment (ThinhKimVo) [inline template/android/link-assets-manifest.json:53]: Is it file auto-generate?

File: template/ios/link-assets-manifest.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    {
+      "path": "src/assets/fonts/Roboto-Thin.ttf",
+      "sha1": "711e666e7f52210ac487c7ed81a0ac68a2b52261"
+    },
+    {
+      "path": "src/assets/fonts/Roboto-ThinItalic.ttf",
+      "sha1": "75ffde032d49005faa643338eda97c311ad7d316"
+    }
+  ]
+}
```
Comment (ThinhKimVo) [inline template/ios/link-assets-manifest.json:53]: Is it file auto-generate?

**Comment by ThinhKimVo** [comment]
> Does all the app use the same `fontFamily`?

We should be defined an example fonts, ex: normal, bold, light. Because this will be change by specific project.

**Review by tuledu — APPROVED** [review]

**Pull request closed without merging**